util/test_timeline.py

```diff
@@ -370,24 +347,36 @@ def run_file(args, timelist):
         'branch': 1,
         'timeline_stopped': True
     }
-    file_started = False
+    started = False
 
     with args.file as file:
+        # If searching for encounters, divert and find start/end first3
```

Comment:
:3

Reply:
Gah. It keeps returning. An undead comment, of *all* things.